### ISSUE TITLE
About the shared Weight!

### ISSUE THREAD

**Issue by yczlab** [fanq15/FSOD-code#51]
Does the RPN/Multi-Relation Detector share parameters between 2 branch or more branch?

**Comment by fanq15** [comment]
Yes, they share weights. You can reference the new codebase for better network details: https://github.com/fanq15/FewX

**Comment by yczlab** [comment]
Thanks. But how to train on voc datasets.



---Original---
